### PR TITLE
RFP kudos-widget

### PR DESCRIPTION
**Pull request by KazanderDad**
Created the RFP for the Kudos widget. Would appreciate your prompt review of this, as we would like to put it out to market ASAP.

### CODE REVIEW

File: RFPs/rfp-1-kudos-widget.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Status: Published
+Category: NEAR Social
+Created: 2023-05-11
+Replaces: 
+Requires: spec-1-kudos.md
+---
+
+# RFP-2: Kudos Widget for NDC
+
+## Summary
```
Comment (KazanderDad) [inline RFPs/rfp-1-kudos-widget.md:12]:
```suggestion
## Summary

```

File: RFPs/rfp-1-kudos-widget.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Request For Proposal
+
+**NDC is looking for proponents to submit their proposals to develop a "Kudos" widget on near.social.**
+  
+## Spec
+  
+Kudos is a widget that will enable people to build on-chain reputation. The widget will allow any verified human to nominate, recieve, and interact with Kudos on Near Social. 
+
+Special attention is required to spam protection in order to ensure that people percieve interacting with Kudos as a very positive experience. Key spam protection concepts include:
+  - setting preferences for who can send me kudos and who can comment on them
```
Comment (KazanderDad) [inline RFPs/rfp-1-kudos-widget.md:22]:
```suggestion
- setting preferences for who can send me kudos and who can comment on them
```

File: RFPs/rfp-1-kudos-widget.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  - allowing recipients to not accept and/or delete unwanted Kudos
```
Comment (KazanderDad) [inline RFPs/rfp-1-kudos-widget.md:23]:
```suggestion
- allowing recipients to not accept and/or delete unwanted Kudos
```

File: RFPs/rfp-1-kudos-widget.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  - requiring that users are i-am-human-certified before they can send or comment on Kudos
```
Comment (KazanderDad) [inline RFPs/rfp-1-kudos-widget.md:24]:
```suggestion
- requiring that users are i-am-human-certified before they can send or comment on Kudos
```

File: RFPs/rfp-1-kudos-widget.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    - Location: Comments on [NDC Gigs](https://near.social/#/neardigitalcollective.near/widget/Gigs)
+    - Deadline: **Tuesday May 23, 2023**
+6. Expected delivery time: **June 23, 2023**
+
+## Selection Criteria
+
+The Governance Working Group will review these proposals and post an Attestation to the various proposals as part of making the selection. ‍ We may also select a runner-up in case the primary proponent does not pass KYC.
+
+Evaluation criteria will include
+- Solution architecture
```
Comment (KazanderDad) [inline RFPs/rfp-1-kudos-widget.md:55]:
```suggestion

- Solution architecture
```

File: RFPs/rfp-1-kudos-widget.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
+---
```
Comment (robert-zaremba) [inline RFPs/rfp-1-kudos-widget.md:1]: this is the second RFP, so the filename should be `rfp-2-kudos-widget.md`.

File: RFPs/rfp-1-kudos-widget.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+- allowing recipients to not accept and/or delete unwanted Kudos
+- requiring that users are i-am-human-certified before they can send or comment on Kudos
+
+Please refer to this link to see the full specification for this RFP: **[spec-1-kudos.md](https://github.com/near-ndc/gwg/blob/main/specs/spec-1-kudos.md)**
+
+## Expected Timeline
+
+1. Proponents post their intent to respond
+    - Location: Comment on [NDC Gigs](https://near.social/#/neardigitalcollective.near/widget/Gigs)
+    - Deadline: **Wednesday May 17, 2023**
```
Comment (robert-zaremba) [inline RFPs/rfp-1-kudos-widget.md:34]: might be too early - let's do until next Friday

File: RFPs/rfp-1-kudos-widget.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+2. Proponents post their questions
+    - Location: Comment on [NDC Gigs](https://near.social/#/neardigitalcollective.near/widget/Gigs)
+    - Deadline: **Wednesday May 17, 2023**
```
Comment (robert-zaremba) [inline RFPs/rfp-1-kudos-widget.md:37]: as above, let's push all dates below accordingly.

**Review by robert-zaremba — APPROVED** [review]
looks good, left few suggestions

**Pull request merged**